src/app/reducers/knowledge.ts

```diff
@@ -0,0 +1,32 @@
+import { compose } from '@ngrx/core';
+import { combineReducers } from '@ngrx/store';
+import { ActionReducer, Action } from '@ngrx/store';
+import { createSelector } from 'reselect';
+import * as knowledge from '../actions/knowledge';
+export const CHANGE = 'CHANGE';
```

Comment:
Create a separate action for this rather than adding a constant value here.

Reply:
I do not know why that `const` CHANGE is exported there, actually it is not used. I just exported it because it was done before.Therefore, It do not make sense to create a separate action for it. I am therefore going to remove it.

Reply:
Also the Action for Knowledge is already there.

Reply:
Removing `const` CHANGE is decreasing coverage.